src/api/app/controllers/webui/package_controller.rb

```diff
@@ -835,11 +838,12 @@ def live_build_log
 
     if @package && !@package.check_source_access?
       flash[:error] = 'Could not access build log'
-      redirect_to action: :show, project: @project.name, package: @package.name
+      redirect_to action: :show, project: @project.name, package: params[:package]
```

Comment:
It might be different ... but actually we need indeed to redirect to the @package.name one here.